### PR TITLE
Migration from sass to scss

### PR DESCRIPTION
**Pull request by A-Ostrovnyy**

### CODE REVIEW

File: src/sass/app.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import 'helpers/index';
```
Comment (zonzujiro) [inline src/sass/app.scss]: Probably you should rename folder to `styles` from `saass`

File: src/sass/components/_about.scss [hunk 2]
```diff
@@ -0,0 +1,151 @@
+.section--about {
+  padding-bottom: 40px;
+
+	.section__sidebar {
+    display: flex;
```
Comment (zonzujiro) [inline src/sass/components/_about.scss]: Pls, apply `prettier`

File: src/sass/components/_hero.scss [hunk 3]
```diff
@@ -1,109 +1,137 @@
...
...roke='%23979797' stroke-width='2' stroke-linejoin='round'/%3E%3Cpath d='M50.5685 34.5043C50.5685 34.5043 53.4041 30.3721 53.4041 25.9179C53.4041 21.4637 50.5685 16.7949 50.5685 16.7949' stroke='%23979797' stroke-width='2' stroke-linejoin='round'/%3E%3Cpath d='M17.4863 34.5043C17.4863 34.5043 14.6507 30.3721 14.6507 25.9179C14.6507 21.4637 17.4863 16.7949 17.4863 16.7949' stroke='%23979797' stroke-width='2' stroke-linejoin='round'/%3E%3Cpath d='M38.2808 28.7607L42.0616 20.1453L45.8425 28.7607' stroke='%23979797' stroke-width='2' stroke-linejoin='bevel'/%3E%3Cpath d='M22.2123 28.7607L25.9932 20.1453L29.774 28.7607' stroke='%23979797' stroke-width='2' stroke-linejoin='bevel'/%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M34.5 34C35.3284 34 36 33.3284 36 32.5C36 31.6716 35.3284 31 34.5 31C33.6716 31 33 31.6716 33 32.5C33 33.3284 33.6716 34 34.5 34Z' fill='%23979797'/%3E%3Cpath d='M19.8493 12.0085L25.0479 1L30.2466 12.0085' stroke='%23979797' stroke-width='2' stroke-linejoin='bevel'/%3E%3Cpath d='M38.7534 12.0085L43.952 1L49.1507 12.0085' stroke='%23979797' stroke-width='2' stroke-linejoin='bevel'/%3E%3Crect x='30' y='11' width='9' height='2' fill='%23979797'/%3E%3C/svg%3E%0A");
```
Comment (zonzujiro) [inline src/sass/components/_hero.scss]: Can you mobe this to separate `svg`?

**Comment by zonzujiro** [comment]
This PR partially resolves #159

File: src/sass/components/_about.scss (hunk 2 above)
Comment (yevhenorlov) [reply src/sass/components/_about.scss]: and `prettier:fix"` script in `package.json` as well

Comment (yevhenorlov) [reply src/sass/components/_about.scss]: done in https://github.com/kottans/kottans.github.io/pull/164

**Comment by yevhenorlov** [comment]
@A-Ostrovnyy I've updated the prettier config, could you please run `npm run prettier:fix` and commit again?

**Pull request merged**